docs/store/datastore.md

```diff
@@ -22,14 +23,15 @@ Data stores are referred to using the schema prefix (e.g. `s3://my-bucket/path`)
 
 ## Storage credentials and parameters
 Data stores might require connection credentials. These can be provided through environment variables 
-or project/job context secrets. The exact credentials depend on the type of the data store and are listed in
-the following table. Each parameter specified can be provided as an environment variable, or as a project-secret that
-has the same key as the name of the parameter.
+or project/job context secrets. The exact credentials depend on the type of the data store. They are 
+listed in the following sections. Each parameter specified can be provided as an environment variable, 
+or as a project-secret that has the same key as the name of the parameter.
 
-MLRun jobs executed remotely run in independent pods, with their own environment. When setting an environment 
+MLRun jobs that are executed remotely run in independent pods, with their own environment. When setting an environment 
 variable in the development environment (for example Jupyter), this has no effect on the executing pods. Therefore, 
-before executing jobs that require access to storage credentials, these need to be provided by assigning environment 
-variables to the MLRun runtime itself, assigning secrets to it, or placing the variables in project-secrets.
+before before executing jobs that require access to storage credentials, these need to be provided by creating a datastore.
```

Comment:
* redundant "before"
* these need to be provided by creating a datastore" - not sure what you meant to say here. how is creating a datastore helping in accessing the credentials..